samples/gallery/build.gradle

```diff
@@ -17,6 +17,9 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$JETBRAINS_KOTLINX_COROUTINES_VERSION"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$JETBRAINS_KOTLINX_COROUTINES_VERSION"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$JETBRAINS_KOTLIN_VERSION"
+    implementation "androidx.compose.foundation:foundation:1.2.0-rc03"
+    implementation 'androidx.compose.ui:ui:1.2.0-rc03'
```

Comment:
This can use the stable (1.2.0) version of the libraries. Generally, we want libraries to use the latest stable version of Compose.

Reply:
Done.